Library/Homebrew/utils/github/api.rb

```diff
@@ -184,7 +184,9 @@ def open_rest(url, data: nil, data_binary_path: nil, request_method: nil, scopes
       # This is a no-op if the user is opting out of using the GitHub API.
       return block_given? ? yield({}) : {} if Homebrew::EnvConfig.no_github_api?
 
-      args = ["--header", "Accept: application/vnd.github+json", "--write-out", "\n%\{http_code}"]
+      # rubocop:disable Style/FormatStringToken
```

Comment:
Can this get a comment explaining the disable?

Reply:
Effectively because it's not a format token - we're passing it to curl, not Ruby's string formatting.

Reply:
> We worked around it by escaping, but meaningless escapes are blocked by the new cop.
> I'll think how to word that.

Yeh, I think just that is fine "not a Ruby format token but a Curl one"